Configuration/Geometry/README.md

```diff
@@ -88,5 +88,6 @@ Several detector combinations have been generated:
 * D63 = T21+C11+M4+I11+O5+F4
 * D64 = T22+C11+M4+I11+O5+F4
 * D65 = T23+C11+M4+I11+O5+F4
+* D66 = T17+C11+M8+I11+O5+F4
```

Comment:
@bsunanda T17 is soon to be decommissioned (next week). I would suggest to use T21 as new baseline instead of T17. @emiglior FYI

Reply:
Changed T17 to T21

Reply:
@mmusich while I agree for new scenarios it's better to use a more modern tracker, what is the intention for the 6 scenarios currently using T17 when it is "decommissioned"?

Reply:
I suppose they should all be migrated to T21. @emiglior is preparing a PR about that.

Reply:
Unless of course some other subsystem explicitly wants to keep T17 around.

Reply:
Let's move to #30512